index.js

```diff
@@ -351,9 +338,16 @@ function addPatternProperties (schema, externalSchema, fullSchema) {
           json += $asString(keys[i]) + ':' + $asBoolean(obj[keys[i]])
       `
     } else {
-      code += `
+      if (options && options.noThrow) {
+        code += `
+        ${addComma}
+        return {error: "Cannot coerce " + obj[keys[i]] + " to ${type}"}
```

Comment:
Why isn't the value of the `error` property an instance of `Error`?

Reply:
Sometimes i wanted to get only object instead of Error improve performance before

Reply:
I would be okay with this change if the error property is an instance of the Error object.